CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.0.11
+
+### Enhancements
+
+* Fix OpenSearch connector
```

Comment:
Nit: the usual pattern we have in the changelog is: 
```
* **Short description** Longer description
```

I think there were a couple that snuck in without that pattern.